server/src/http/feature_refresher.rs

```diff
@@ -449,6 +465,36 @@ impl FeatureRefresher {
                         });
     }
 
+    async fn handle_client_features_delta_updated(
+        &self,
+        refresh_token: &EdgeToken,
+        delta: ClientFeaturesDelta,
+        etag: Option<EntityTag>,
+    ) {
+        debug!("Got updated client features delta. Updating features with {etag:?}");
+        let key = cache_key(refresh_token);
+        self.features_cache.apply_delta(key.clone(), &delta);
+        self.update_last_refresh(
+            refresh_token,
+            etag,
+            self.features_cache.get(&key).unwrap().features.len(),
```

Comment:
Is this unwrap safe?

Reply:
Should be, the line 476 should guarantee that cache has value with that key.